### ISSUE TITLE
Schema does not exist error when loading Functions virtual folder

### ISSUE THREAD

**Issue by knolee** [Borvik/vscode-postgres#161]
Hi, when I try to expand the functions directory in the explorer, I get the following error:

![image](https://user-images.githubusercontent.com/49707796/103988669-9bdb7e80-5132-11eb-8c4c-e5d28023fa0b.png)

Troubleshooting attempted:

- Restarting VSCode
- removing and re-adding functions folder

Versions: 

- PostgreSQL 1.1.14
- Visual Studio Code: 1.52.1
- Ubuntu 20.04

**Comment by Borvik** [comment]
Ok, so when expanding the `Functions` under a schema, one of two db queries fires, depending on the version of the PostgreSQL server you are connected to.

The version you listed is, I'm sure, for the version of this plugin - not actually the server version.

I have a thought as to the issue (may be related to #33), but if you are up for a little back-and-forth debugging maybe we can solve this.

If you could send your PostgreSQL server version, I will format a couple of queries to run to see which will work and which won't.

**Comment by knolee** [comment]
Hi sure, my PostgreSQL server version is 12.5. My apologies for not including it in the original issue.

**Comment by Borvik** [comment]
No worries.

The actual results (rows) of the queries aren't as important to me - rather than a success/failure occurred.  Success measured by the fact you can see a list of your functions.

Query 1 (will most likely fail):
```sql
SELECT
  n.nspname as "schema",
  p.proname as "name",
  d.description,
  pg_catalog.pg_get_function_result(p.oid) as "result_type",
  pg_catalog.pg_get_function_arguments(p.oid) as "argument_types",
  CASE
    WHEN p.prokind = 'a' THEN 'agg'
    WHEN p.prokind = 'w' THEN 'window'
    WHEN p.prorettype = 'pg_catalog.trigger'::pg_catalog.regtype THEN 'trigger'
    ELSE 'normal'
  END as "type"
FROM pg_catalog.pg_proc p
  LEFT JOIN pg_catalog.pg_namespace n ON n.oid = p.pronamespace
  LEFT JOIN pg_catalog.pg_description d ON p.oid = d.objoid
WHERE
  n.nspname = 'Device'
  AND p.prorettype <> 'pg_catalog.trigger'::pg_catalog.regtype
  AND has_schema_privilege(quote_ident(n.nspname), 'USAGE') = true
  AND has_function_privilege(p.oid, 'execute') = true
ORDER BY 1, 2, 4;
```

Query 2 (_probably_ succeed):
```sql
SELECT
  n.nspname as "schema",
  p.proname as "name",
  d.description,
  pg_catalog.pg_get_function_result(p.oid) as "result_type",
  pg_catalog.pg_get_function_arguments(p.oid) as "argument_types",
  CASE
    WHEN p.prokind = 'a' THEN 'agg'
    WHEN p.prokind = 'w' THEN 'window'
    WHEN p.prorettype = 'pg_catalog.trigger'::pg_catalog.regtype THEN 'trigger'
    ELSE 'normal'
  END as "type"
FROM pg_catalog.pg_proc p
  LEFT JOIN pg_catalog.pg_namespace n ON n.oid = p.pronamespace
  LEFT JOIN pg_catalog.pg_description d ON p.oid = d.objoid
WHERE
  n.nspname = 'Device'
  AND p.prorettype <> 'pg_catalog.trigger'::pg_catalog.regtype
  AND has_schema_privilege(n.oid, 'USAGE') = true
  AND has_function_privilege(p.oid, 'execute') = true
ORDER BY 1, 2, 4;
```

**Comment by knolee** [comment]
Yup, can confirm. The 1st query failed (same "Schema does not exist" error) while the 2nd query worked (returned result set with functions).

**Comment by Borvik** [comment]
Perfect.

Follow-up - when writing a query (language `Postgres`) and a server is selected - do you get intellisense hints about the tables and columns?

**Comment by knolee** [comment]
Nope, only schema hints show up - table/column names do not.

**Comment by Borvik** [comment]
published new version 1.1.15 more intellisense should be available, and the function tree should now work

**Comment by knolee** [comment]
works great, thank you!
